src/typing/flow_error.ml

```diff
@@ -165,6 +165,7 @@ type error_message =
       null_loc: Loc.t;
       falsy_loc: Loc.t;
     }
+  | EUnknownProperty of Loc.t * string
```

Comment:
It would probably be a good idea to copy the implementation for `EStrictLookupFailed` for `ENonstrictLookupFailed`. We’ll just output an `UnknownPropertyInBranchTest` lint.